packages/interactor/test/definitions/check-box.test.ts

```diff
@@ -215,6 +215,24 @@ describe('@bigtest/interactor', () => {
       });
     });
 
+    describe('filter `indeterminate`', () => {
+      it('filters `input` tags by whether they are indeterminate', async () => {
+        dom(`
+          <p>
+            <label for="accept-field">Accept</label>
+            <input type="checkbox" id="accept-field"/>
+          </p>
+        `);
+
+        // NOTE: No browser currently supports indeterminate as an attribute. It must be set via JavaScript.
+        // NOTE: See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes
+        await CheckBox('Accept').perform(e => e.indeterminate = true)
+
+        await expect(CheckBox('Accept', { indeterminate: true }).exists()).resolves.toBeUndefined();
+        await expect(CheckBox('Accept', { indeterminate: false }).exists()).rejects.toHaveProperty('name', 'NoSuchElementError');
```

Comment:
For sanity check, can we also assert that both `checked` and `unchecked` are false?

Reply:
The `checked` could be `true` or `false`. Think about it as the `indeterminate` state has a higher priority to UI view. Maybe we have to check `indeterminate == false` in the `check` filter?

Reply:
![image](https://user-images.githubusercontent.com/6397708/112444545-f9658b80-8d6f-11eb-822a-a77203d04b12.png)

Reply:
Hm... Looks like we must reset `indeterminate` state on any `check/uncheck/toggle/click` actions. Ok, I'll fix it

Reply:
Where are we on this?

Reply:
Because I think this should be ready to go, right?

Reply:
Yep, I remember about it. I just was a little busy with datepicker.

Reply:
Ok, just wanted to make sure I didn't let it slip through the cracks